src/content/getting-started/designers/designers.mdx

```diff
@@ -50,12 +49,12 @@ Carbon empowers designers to rapidly build beautiful and accessible experiences.
 ## Get the kit
 
 1. [**Install Sketch**](https://www.sketchapp.com/). To design with Carbon you must have the most recent version of Sketch installed.
-2. [**Add libraries**](#resources). Designing with Carbon requires subscribing to two libraries: The IBM Design Language kit and the Carbon kit. Click the tiles above to subscribe to each. Both libraries are required.
```

Comment:
We gotta avoid bolding links. If we wanna include a link, how about: 
1. **Install Sketch**. To design with Carbon you must have the most recent version of [Sketch](https://www.sketchapp.com/) installed.
![image](https://user-images.githubusercontent.com/15144993/55108304-5656da00-50a1-11e9-982a-b17aa8995180.png)

Reply:
Ok! I'll fix and push. Thank you!